src/mbgl/gl/vertex_buffer.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <mbgl/gl/bytes.hpp>
+#include <mbgl/gl/object.hpp>
+
+namespace mbgl {
+namespace gl {
+
+template <class Vertex>
+class VertexBuffer {
+public:
+    static constexpr std::size_t vertexSize = sizeof(Vertex);
```

Comment:
Could we use `using vertexSize = std::integral_constant<std::size_t, sizeof(Vertex)>;` to avoid static data members that need initialization?